entity-detail.js

```diff
@@ -0,0 +1,101 @@
+var rowData;
+var table;
+var akasTable;
+$('document').ready(function(){
+  var dataResultLength=data.result.length;
+  if((dataResultLength == 1) && !(data.result[0].meta)){
```

Comment:
No need to use 
```js
if(dataResultLength == 1)
```
It can be rewritten as below:
```js
if ( data.result.meta ){
  //functionality
}
else{
  //Nothing to show
}
```